scripts/who/gho/svNames/who_gho_sv_name.py

```diff
@@ -0,0 +1,131 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import json
+
+from absl import app
+
+INDICATOR_MAP_FILE = "indicator_map.json"
```

Comment:
Where do these files come from?  Can we check them in too?

Also, could we have a few words in a README on how to run the tool with right inputs?

So if someone other than you needs to repro this in future, they should have enough state here to do so :)

Reply:
indicator_map.json is generated in the script. But it uses the who_properties.mcf file that is in google3 to generate the map .. am I supposed to copy that file here?

Reply:
No, instead I would provide instructions on doing that. Like:

This script edits the `who_properties.mcf` that is checked into the schema (and google3) repo.  Follow the instructions below to update it.

```
WHO_PROPS_DIR=<directory-of-g4-client>/third_party/datacommons/schema/core

cd scripts/who/gho/svNames
cp $WHO_PROPS_DIR/who_properties.mcf .
# Instructions to run this script
cp who_properties.mcf $WHO_PROPS_DIR
```

Reply:
done